src/styles/main.scss

```diff
@@ -1 +1,58 @@
-// put code here
+* {
+  margin: 0;
```

Comment:
Don't use any rules inside `*` selector instead of `box-sizing`. Place `margin` everywhere you need but not do it globally.